src/gendersprachekorrigieren.ts

```diff
@@ -26,34 +31,54 @@ class BeGoneSettingsHelper {
         return settings.filterliste === "Blacklist";
     }
 
-    public static whiteliststring(settings: BeGoneSettings): string {
-        return settings.whitelist ? settings.whitelist.replace(/(\r\n|\n|\r)/gm, "|") : "";
+    public static whitelistRegexp(settings: BeGoneSettings): RegExp {
+        try {
+            let res = urlFilterListToRegex(settings.whitelist);
+            res.test("test");
+            return res;
+        } catch (e) {
+            return RegExp("");
```

Comment:
Falls das Nutzy invalide Eingaben macht, wird es als fallback ignoriert. In `options.ts` wird dann im "#whitelist-error" eine Fehlermeldung ausgegeben.

Ist mir mal passiert, ich habe mich dann gewundert wieso es nicht geht.